plugins/main/public/components/management/cluster/components/configuration_cards.tsx

```diff
@@ -0,0 +1,196 @@
+    <EuiFlexGroup direction='column' gutterSize='s'>
+      <EuiFlexItem>
+        <EuiFlexGroup responsive={false} alignItems='center' gutterSize='s'>
+          <EuiFlexItem grow={false}>
+            <EuiToolTip content='Go back' position='bottom'>
+              <EuiButtonIcon
+                color='primary'
+                size='m'
+                display='empty'
+                iconType='arrowLeft'
+                aria-label='Back'
+                onClick={goBack}
+              />
+            </EuiToolTip>
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiTitle>
+              <h2>Overview</h2>
+            </EuiTitle>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      </EuiFlexItem>
+      <EuiFlexItem>
+        <EuiFlexGroup responsive alignItems='flexStart' gutterSize='s'>
+          <EuiFlexItem
+            grow={false}
+            style={{
+              minWidth: '500px',
+              overflow: 'hidden',
+              height: '100%',
+            }}
+          >
+            {results?.hits?.total > 0 ? (
+              <DashboardByRenderer
+                input={{
+                  viewMode: ViewMode.VIEW,
+                  panels: getDashboardConfigurationPanels(indexPatternId),
+                  isFullScreenMode: false,
+                  filters: filters,
+                  useMargins: true,
+                  id: 'ct-dashboard-configuration-tab',
+                  timeRange: {
+                    from: searchBarProps.dateRangeFrom,
+                    to: searchBarProps.dateRangeTo,
+                  },
+                  title: 'Cluster configuration dashboard',
+                  description: 'Dashboard of the Cluster configuration',
+                  query: searchBarProps.query,
+                  refreshConfig: {
+                    pause: false,
+                    value: 15,
+                  },
+                  hidePanelTitles: false,
+                }}
+              />
+            ) : (
+              <DiscoverNoResults
+                message='There are no results for selected time range. Try another
+            one.'
+              />
+            )}
+          </EuiFlexItem>
+          <EuiFlexItem style={{ padding: '8px 0 0 0', height: '364px' }}>
+            <EuiCard
+              textAlign='left'
+              title={
+                <EuiTitle size='s'>
+                  <h2>Configuration</h2>
+                </EuiTitle>
+              }
+            >
+              <EuiSpacer size='m' />
+              <EuiDescriptionList
+                type='column'
+                compressed={true}
+                align='left'
+                listItems={[
+                  {
+                    title: (
+                      <span style={{ fontWeight: 400, fontSize: '1rem' }}>
+                        Disabled
+                      </span>
+                    ),
+                    description: configuration?.disabled ? 'true' : 'false',
```

Comment:
**nitpick**: the value is rendered without mapping it: https://github.com/wazuh/wazuh-dashboard-plugins/pull/6561/files#diff-788779c8743bae56c6be423eee000dbd0de5c8616f5743e49b6a4e9afce3b73fL291

Reply:
The rendering of this value is fixed and the `listItems` of `EuiDescriptionList` are improved so as not to repeat code.